### PR TITLE
Fixes middleware invocation arguments use by legacy apply update code path

### PR DESCRIPTION
**Pull request by tmat**
Fixes https://github.com/dotnet/AspNetCore-ManualTests/issues/3306

Regressed by https://github.com/dotnet/sdk/pull/44539

### CODE REVIEW

**Review by phil-allen-msft — APPROVED** [review]

File: src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js [hunk 1]
```diff
@@ -159,8 +159,13 @@ setTimeout(async function () {
       });
     }
 
+    let body = JSON.stringify({
+      "id": deltas[0].sequenceId,
+      "deltas": deltas
+    });
+
```
Comment (SteveSandersonMS) [inline src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js:166]: Nitpick on syntax, hope that's OK:

```suggestion
    const body = JSON.stringify({
      id: deltas[0].sequenceId,
      deltas: deltas
    });
```

File: src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js [hunk 2]
```diff
@@ -159,8 +159,13 @@ setTimeout(async function () {
       });
     }
 
+    let body = JSON.stringify({
+      "id": deltas[0].sequenceId,
```
Comment (SteveSandersonMS) [inline src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js:163]: Do we know for certain that `deltas` always contains at least one entry?

File: src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js (hunk 1 above)
Comment (tmat) [reply src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js]: Sure, thanks for the suggestion.

File: src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js (hunk 2 above)
Comment (tmat) [reply src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js]: The client shouldn't be sending updates with empty deltas.

**Comment by marcpopMSFT** [comment]
Known arm64 timeout

**Pull request merged**